src/librepgp/packet-print.c

```diff
@@ -292,16 +294,19 @@ print_block(int indent, const char *name, const uint8_t *str, size_t length)
 }
 
 /* return the number of bits in the public key */
-static int
+static size_t
```

Comment:
This function is used to print information about the key, and I think users would like to see '2048' bits instead of more correct 2045 or 2047.

Reply:
argh... yes, your right